docs/data/sdks/ios-swift/migration.md

```diff
@@ -522,8 +522,8 @@ To install your custom plugin, use `add()` with your custom plugin as parameter.
 | --- | --- | --- |
 | Package | AmplitudeSwift | [Amplitude](https://cocoapods.org/pods/Amplitude-iOS) |
 | Configuration | Configuration is implemented by the configuration object. Configurations need to be passed into Amplitude Object during initialization. [More configurations](../../ios/#configuration). Latest iOS SDK support more configurations. Check [here](../#configuration) for more details. | Support explicity setter methods. [More configurations](../../ios/#configuration) |
-| Logger Provider | ConsoleLogger() by default. Fully customizable. | AMPLITUDE_LOG, config throught marcro. |
+| Logger Provider | ConsoleLogger() by default. Fully customizable. | AMPLITUDE_LOG, config through macro. |
```

Comment:
@yuhao900914 @liuyang1520 can you check if the fix makes sense? this is my best guess

Reply:
That's right. Thanks.